migration/20221008130810-transaction-card-table.js

```diff
@@ -12,7 +12,7 @@ module.exports = class transactionCardTable20221008130810 {
                 \`gsm\` varchar(50) null,
                 \`currency_id\` char(36) null,
                 constraint \`pk_transaction_card\` primary key (\`id\`),
-                constraint \`fk_transaction_card_currency\` foreign key (\`currency_id\`) references \`currency\` (\`id\`) on update cascade on delete set null
+                constraint \`fk_transaction_card_currency\` foreign key (\`currency_id\`) references \`currency\` (\`id\`) on update cascade on delete restrict
```

Comment:
I made the deletion restricted. @RamaMahjoub

Reply:
👍🏻